lighthouse-cli/index.js

```diff
@@ -62,6 +62,7 @@ const cli = yargs
     'list-all-audits': 'Prints a list of all available audits and exits',
     'list-trace-categories': 'Prints a list of all required trace categories and exits',
     'config-path': 'The path to the config JSON.',
+    'launch-chrome': 'Launch chrome automatically when accessing port 9222 fails',
```

Comment:
specifying "false" in a cmd line flag is really awkward to me, so I really prefer when the flag itself is only used when you're setting it as truthy.

Basically, apply a flag (with no explicit boolean value) in order to bypass a default.

favorites right now: `skip-autolaunch` `avoid-autolaunch`